flink-kubernetes-operator/src/test/java/org/apache/flink/kubernetes/operator/TestUtils.java

```diff
@@ -421,6 +427,28 @@ public static void setEnv(Map<String, String> newEnv) {
         }
     }
 
+    public static <T extends AbstractFlinkResource<?, ?>> MetricManager<T> createTestMetricManager(
```

Comment:
nit: There's a MetricListener utility class for this.

Reply:
The MetricListener doesn't expose the registry and also it hardcodes a base group, so I had to remove it from our tests